src/services/legal-services.ts

```diff
@@ -50,59 +42,63 @@ export default class LegalApiMixin extends Vue {
 
   /**
    * Updates an existing filing.
-   * @param filing the object body of the request
+   * @param businessId the id of the business to update
+   * @param filingId the id of the filing to update
+   * @param filing the object body of the filing
    * @param isDraft boolean indicating whether to save draft or complete the filing
    * @returns a promise to return the updated filing
    */
-  async updateFiling (
+  static async updateFiling (
+    businessId: string,
+    filingId: number,
     filing: CorrectionFilingIF | AlterationFilingIF | ChgRegistrationFilingIF | ConversionFilingIF,
     isDraft: boolean
   ): Promise<any> {
-    if (!filing) throw new Error('updateFiling(), invalid filing')
```

Comment:
These parameter checks should never fail at runtime. And during development, TS and typings should identify parameter errors already. So I removed these (also below).